packages/create-cloudflare/src/helpers/args.ts

```diff
@@ -270,11 +310,7 @@ export const parseArgs = async (argv: string[]): Promise<Partial<C3Args>> => {
 
 	if (options) {
 		for (const { name, alias, ...props } of options) {
-			const values =
-				typeof props.values === "function"
-					? await props.values(await yargsObj.argv)
-					: props.values;
-			yargsObj.option(name, { values, ...props });
```

Comment:
This is not working as intentional:
- The `value` is always overridden by `props.value`
- `yargsObj.option` do not support `values` and so it is always ignored